### PR TITLE
Bug 1383690 added oc patch to cli reference

### PR DESCRIPTION
**Pull request by bfallonf**
As per:

https://bugzilla.redhat.com/show_bug.cgi?id=1383690

Will ask for an ack in the BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
Ack given. Peer review plz?

@adellape @ahardin-rh

File: cli_reference/basic_cli_operations.adoc [hunk 1]
```diff
@@ -467,6 +467,20 @@ $ oc run NAME --image=<image> \
     [options]
 ----
 
+=== patch
+Updates one or more fields of an object using strategic merge patch:
+----
```
Comment (ahardin-rh) [inline cli_reference/basic_cli_operations.adoc]: add a space before the code block to ensure that the formatting renders properly

Comment (bfallonf) [inline cli_reference/basic_cli_operations.adoc]: Hmm the rest of the file is like this, and this is the first i'm hearing of this being a thing. I'll leave for now, but maybe I'll do some comparisons one day...

**Comment by bfallonf** [comment]
Thanks @ahardin-rh I'll merge.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../cli_reference/basic_cli_operations.adoc#cli-reference-basic-cli-operations[Developer CLI Operations]
|Added information on `oc patch`.
%

**Pull request merged**